### ISSUE TITLE
Enhance "DeletePackage"

### ISSUE THREAD

**Issue by aplteam** [aplteam/Tatin#175]
The syntax 

```
]DeletePackage Foo
```

should be allowed. If there is just one version of that package published, it should be deleted after asking for confirmation.

Same for 

```
]DeletePackage groupName-Foo
```

in case the package name is used in more than one group.

Note that currently 

```
]DeletePackage Foo
```

results in `unauthorized request` no matter what, which is wrong anyway. Such a message makes only sense in case one tries to delete a package of a group she has not access to.

**Comment by aplteam** [comment]
It might well be a good idea to delete ALL packages when there are more than one published.

**Comment by aplteam** [comment]
> It might well be a good idea to delete ALL packages when there are more than one published.

After questioning the user when it was issued as a user command.

With the API function, the syntax 

```
⎕SE.Tatin.DeletePackage '[local]Foo.*
```

might be supported.

This is particularly important for development servers, when a potentially significant number of beta versions of a package is released, all of which need to be deleted once the final version of the package is published elsewhere.

**Comment by aplteam** [comment]
Implemented in 0.105.0
